inc/class-command.php

```diff
@@ -65,6 +65,15 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 			case 'stop':
 				return $this->stop( $input, $output );
 
+			case 'provision':
+				return $this->provision( $input, $output );
+
+			case 'ssh':
```

Comment:
Let's rename this to `shell`, to match `local-server`.

Reply:
Moved to another PR to make the changelog cleaner.